template/template.go

```diff
@@ -271,18 +283,21 @@ func (te Expander) Expand() (result string, resultErr error) {
 			var ok bool
 			resultErr, ok = r.(error)
 			if !ok {
+				expandFailures.Inc()
```

Comment:
you can remove the other `expandFailures.Inc()` if you move add this at the end of the defer function 

```go
if resultErr != nil {
  expandFailures.Inc()
}
```